pages/documentation/glossary.md

```diff
@@ -7,5 +7,10 @@ layout: default
 <span class="heading" id="exporter">**Exporter:**</span>  
 An exporter is a class responsible for converting an [IIDM]() model and writing it to a file.
 
-<span class="heading" id="importer">**Importer:**</span>  
+<span class="heading" id="importer">**Importer:**</span>
+<a id="importer"/>  
```

Comment:
Looking to the license.md file, it's not needed to add an `anchor`. Could you check it's really necessary and remove the two anchors you add if not?

Reply:
Tested on Chrome web browser: if you do not add the anchor the relative link inside the page does not work. If tested it building a very long page, with "tso" section at the end. If we do not add the `<a>`  element, then the link `glossary.md#tso` leaves you at the top of the page. After adding the `<a>`, the browser scrolls automatically until the text  where the anchor is located is visible.

Reply:
This link https://www.powsybl.org/pages/overview/license#51 works for me on Firefox, Edge (chromium) and Chromium under Linux. If we put an anchor, we shoud remove the span. Need more check to decide what to do :)